squiRL/common/agents.py

```diff
@@ -89,11 +90,37 @@ def play_step(
         action = self.get_action(net)
 
         # do step in the environment
-        new_obs, reward, done, _ = self.env.step(action)
-        exp = Experience(self.obs, action, reward, done, new_obs)
-        self.replay_buffer.append(exp)
+        self.env.act(action)
+        reward, new_obs, first = self.env.observe()
+
+        s = dict(
+            zip(Experience._fields,
+                (self.obs, action, reward, first, new_obs)))
+        step = {
+            k: {e_k: []
+                for e_k in range(self.n_envs)}
+            for k in self.rollouts.keys()
```

Comment:
May be reading this wrong, but I'm under the impression that the keys of rollout and the fields of Experience should be the same? In the interest of readability maybe it is worth using which ever is more static?

Reply:
Good idea. I replaced other instances of this as well.